src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/SchemaGenerator.cs

```diff
@@ -195,6 +195,7 @@ private OpenApiSchema GenerateObjectSchema(SerializerMetadata serializerMetadata
 
             foreach (var serializerPropertyMetadata in serializerMetadata.Properties)
             {
+                if (serializerPropertyMetadata.MemberInfo==null) continue;
```

Comment:
this is probably the important bit, need to verify on a webapp where memberinfo is null though